bindings/gumjs/gumduksymbol.c

```diff
@@ -29,6 +30,7 @@ static const duk_function_list_entry gumjs_symbol_module_functions[] =
   { "getFunctionByName", gumjs_symbol_get_function_by_name, 1 },
   { "findFunctionsNamed", gumjs_symbol_find_functions_named, 1 },
   { "findFunctionsMatching", gumjs_symbol_find_functions_matching, 1 },
+  { "loadModule", gumjs_load_module, 1 },
```

Comment:
```suggestion
  { "load", gumjs_symbol_load, 1 },
```

I.e. `DebugSymbol.load()`.

We will also need to implement this in the V8 runtime.